public/app/features/browse-dashboards/components/SearchView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { parseRouteParams } from 'app/features/search/utils';
+
+interface SearchViewProps {
+  searchState: ReturnType<typeof parseRouteParams>;
+}
+
+export default function SearchView({ searchState }: SearchViewProps) {
```

Comment:
minor nit: think we [prefer named exports](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#exports)